pkg/loadbalancer/experimental/test_utils.go

```diff
@@ -300,7 +301,31 @@ func DumpLBMaps(lbmaps LBMaps, feAddr loadbalancer.L3n4Addr, sanitizeIDs bool, c
 		panic(err)
 	}
 
+	maglevCB := func(key lbmap.MaglevOuterKey, _ lbmap.MaglevOuterVal, _ lbmap.MaglevInnerKey, innerValue *lbmap.MaglevInnerVal, _ bool) {
+		key = lbmap.MaglevOuterKey{
+			RevNatID: byteorder.NetworkToHost16(key.RevNatID),
+		}
+		trunc := innerValue.BackendIDs[:min(10, len(innerValue.BackendIDs))]
+		tail := ""
+		if len(innerValue.BackendIDs) > len(trunc) {
+			tail = "..."
+		}
+		var ids []string
+		for _, id := range trunc {
+			ids = append(ids, sanitizeID(id, sanitizeIDs))
+		}
+		maglev = append(maglev, fmt.Sprintf("MAGLEV: ID=%s INNER=[%s%s]",
```

Comment:
That's a nice idea! But in tests we typically sanitize IDs to `zero` or `non-zero` (see `/testdata`), so we probably also should not rely on their order `1, 2, 3`, right? Perhaps sorting by counts rather than by IDs is the way to go?

Reply:
> Perhaps sorting by counts rather than by IDs is the way to go?

Done.